test/test-files/ssl/example.test/README.md

```diff
@@ -6,6 +6,8 @@ These certificates for usage with TLS have been generated via the [Smallstep `st
 
 `Certificate Details` sections are the output of: `step certificate inspect cert.<key type>.pem`.
 
+Each certificate except for the wildcard one, have the SANs(Subject Alternative Name) `example.test` and `mail.example.test`.
```

Comment:
Why `example.test` and not `example.com` etc. ?

Reply:
Purely for testing purposes, as the TLD is intended for.

Because `example.com` will resolve via external DNS queries, but `external.test` will not, it's a reserved TLD for internal use among others. My own DNS resolver can be configured to accept it, but public ones will reject it.

I find that helpful for testing as when it does try to query a public resolver for whatever reason I know that it'll fail there too and not mislead me in the logs when troubleshooting :sweat_smile: